dataactbroker/handlers/fileHandler.py

```diff
@@ -1704,6 +1730,13 @@ def fabs_derivations(obj):
     else:
         logger.error("CFDA title not found for CFDA number %s", obj['cfda_number'])
 
+    # deriving awarding agency name
+    if 'awarding_agency_code' in obj.values() and obj['awarding_agency_code']:
```

Comment:
This is always going to be false, we can just remove the entire thing. Also, we just noticed that there's a print a few lines down from this. Can you just remove it while you're in there?